<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanConfigDialog</class>
 <widget class="QDialog" name="CanConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CAN配置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="canConfigGroupBox">
     <property name="title">
      <string>CAN参数配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>设备：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="devComboBox"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>索引：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="indexComboBox">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="can0Tab">
         <attribute name="title">
          <string>CAN0</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>验收码：0x</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="accessCode0LineEdit">
            <property name="text">
             <string>00000000</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>屏蔽码：0x</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="maskCode0LineEdit">
            <property name="text">
             <string>FFFFFFFF</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>波特率：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="baudrate0ComboBox">
            <item>
             <property name="text">
              <string>1000 Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>800 Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>500 Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>250 Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>125 Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100 Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50 Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20 Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10 Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5 Kbps</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>定时器0:0x</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="timer00LineEdit">
            <property name="text">
             <string>00</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>定时器1:0x</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="timer01LineEdit">
            <property name="text">
             <string>14</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>滤波方式：</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="filterType0ComboBox">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>双滤波</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>单滤波</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>工作模式：</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QComboBox" name="workMode0ComboBox">
            <item>
             <property name="text">
              <string>正常</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>只听</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="can1Tab">
         <attribute name="title">
          <string>CAN1</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_30">
            <property name="text">
             <string>验收码：0x</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="accessCode1LineEdit">
            <property name="text">
             <string>00000000</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>屏蔽码：0x</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="maskCode1LineEdit">
            <property name="text">
             <string>FFFFFFFF</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>波特率：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="baudrate1ComboBox">
            <item>
             <property name="text">
              <string>1000 Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>800 Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>500 Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>250 Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>125 Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100 Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50 Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20 Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10 Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5 Kbps</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>定时器0:0x</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="timer10LineEdit">
            <property name="text">
             <string>00</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>定时器1:0x</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="timer11LineEdit">
            <property name="text">
             <string>14</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>滤波方式：</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="filterType1ComboBox">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>双滤波</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>单滤波</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>工作模式：</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QComboBox" name="workMode1ComboBox">
            <item>
             <property name="text">
              <string>正常</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>只听</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string> 初始CANID配置 - 通道 0</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>车速CANID：0x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="speedCanidLineEdit">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>18FEF100</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="brakeCheckBox">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>刹车-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="minRpmLineEdit">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="maxLength">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>rpm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="clutchCheckBox">
          <property name="text">
           <string>离合</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>数据：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="speedDataLineEdit">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>00 00 00 00 00 00 00 00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>转速CANID：0x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="rpmCanidLineEdit">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0CF00400</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>数据：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="rpmDataLineEdit">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>00 00 00 00 00 00 00 00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CanConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CanConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
